guides/use-microprofile-lra.adoc

```diff
@@ -0,0 +1,165 @@
+= Use MicroProfile LRA in WildFly
```

Comment:
Also very minor, it could be good to make the headings consistent by choosing between either 

- <Verb>-ing <rest of the sentence> (example: Using Microprofile LRA in WildFly) vs. 
- <Verb> <rest of the sentence> (example: Use Microprofile LRA in WildFly)

Reply:
All titles follow `Verb` -ing + rest of the sentence now